ebpfsvc/svcmain.cpp

```diff
@@ -12,23 +12,23 @@ SERVICE_STATUS_HANDLE ebpf_service_status_handle;
 HANDLE ebpf_service_stop_event_handle = nullptr;
 
 void WINAPI
-service_control_handler(DWORD ctrl);
+service_control_handler(unsigned long ctrl);
 void
-service_report_event(PTSTR function);
+service_report_event( wchar_t* function);
```

Comment:
```suggestion
service_report_event(_In_z_ wchar_t* function);
```
can we add SAL annotation?